### PR TITLE
Fix typo in readme

### PR DESCRIPTION
**Pull request by danielhasan1**
fix typo

### CODE REVIEW

**Review by agronholm — APPROVED** [review]
It fixes grammatical errors, not typos. But the fixes look alright.

**Comment by danielhasan1** [comment]
> It fixes grammatical errors, not typos. But the fixes look alright.

Thanks for the approval yeah I was checking it out and saw some sentences could be improved.

File: README.rst [hunk 1]
```diff
@@ -12,12 +12,12 @@
...
 
 AnyIO is an asynchronous networking and concurrency library that works on top of either asyncio_ or
-trio_. It implements trio-like `structured concurrency`_ (SC) on top of asyncio, and works in harmony
+trio_. It implements trio-like `structured concurrency`_ (SC) on top of asyncio and works in harmony
 with the native SC of trio itself.
 
 Applications and libraries written against AnyIO's API will run unmodified on either asyncio_ or
 trio_. AnyIO can also be adopted into a library or application incrementally – bit by bit, no full
-refactoring necessary. It will blend in with native libraries of your chosen backend.
+refactoring is necessary. It will blend in with the native libraries of your chosen backend.
```
Comment (graingert) [inline README.rst:20]: `no full refactoring necessary` is correct, changing to `no refactoring is necessary` changes the tone of the statement

Comment (agronholm) [inline README.rst:20]:
```suggestion
refactoring necessary. It will blend in with the native libraries of your chosen backend.
```

Comment (danielhasan1) [reply README.rst]: oh okay. thanks for clearing that out.

**Pull request merged**